statick_tool/plugins/discovery/ros_discovery_plugin.py

```diff
@@ -29,6 +29,7 @@ def deep_get(
 
         Copied from:
 
+
```

Comment:
this second blank line is weird, but if it was needed to pass docformatter i'm fine with it.

Reply:
I agree. This is what comes of docformatter and black with our current dependency versions.